### DISCUSSION TITLE
纯CSS打造，LOGO动画

### DISCUSSION THREAD

**Discussion by wenkeming** [ikun-svelte/ikun-ui#30] [Show and tell]
点击查看：[VUE SFC](https://play.vuejs.org/#eNq1WG2PozYQ/isup1WSU0jIZpPdpXvt9aqT2n5oT71+qcQXB0ziW4IRNvvSVf57Z2wghkCSvfYUCfB4PH7m7cHkxfkpyyYPBXN8506GOc8UkUwV2Q9ByreZyBV5ITmLyY7EudiSAagO6qn7Iv2F0aicmkxh7G5AgAYbWh9E9NzQWoHgUIsmSVMLBKVWKFKpSJbQZ/IOAQ1jmkg2CtK4SEPFRUo2NI0S9gk0hqOXICVaefJAk4LBku/2o++DdBekd1PjLjgKA8W2oKAYjAi5k8yY5NG7wOGAJHCIHyZUShi/oKVd4GhVUK6cJlNbgg62JOidkcDeZgczWhVKwW7vw4SH97DD3hPY5eXFeP0jGXxWIhsQnwxwakB2u7upWQpm7qaWCzCU6jnBxzcI34RDSI57+hA+0OMPGAhCHnmkNj6ZLbzsSQu2NF9z0LoEAaGFElrqusonc1k+IyJXKqoK6ZOMFrIMqt5tgrN6y5ZiXqQpT9dG1SDXaiWEGsGG8fUGtqsFKxrer3NRpJEbikTkPnlze3trpkQeMRCkIjX+GIGb04jjngvvQovDIpe4MBM8VSzXsohLxOeTVSLC+4bzuLV2nng2XH8jHliuQXdgim9KZcivDr8zdpSEyo35evJFihSaTK8NnFBsM56w/I8MUyIDxyd6BuegTsTjb1qm8oKNK3m4YeF9h/yLfEJZ4HzKmWT5Awucek6BP0yZ6Y+ff2dP8FxPbkVUJKB9ZPJPJkVSIEaj9gH8BdiWnkb7q+5hyO1f8uOTYqmsnEKgqLnT+oEDzfzzEdf3cOeTuV4H8YQo2sSCTNVoV6thqx6t9KF/7F5rNEndIpNKu4RRleNlWX1WQe5FPd1EwGOayljkW5gRUPhsOJsvIrYelfNVhdWdRQgFDgQTaKwmkweaD6HnRoRD9aRcMajgFQ/dFfuHs3zoTZY3Y+J6k8ViTLzJ5RJuMxiMyoVW31UbY+XGkCufbHgUsdSUau28768YwMaysmQ0hmYp4wKFDKmFMAwGB0GgK10nVRD6Q3i5rEVHupqQ6dtWI3v6N/Ogn99Ou6GXOOsMuCLnOta4Sjdyd4JurPRYudjyyAWC4MoFMxv1P6ek6YAd5zJ4VqQ66k9Dgqi0nTrXZ/ek0wmLv4HP7+/Zc5zTLZN1rZduexdjZGswgMhLYQu8fsT+/Xvojbr7C/bA2+UCLF0vLk7amXkX/ZaagJuxeQXsrpiXOOe49MbrwVnhOg+VronXw/paVO4hLOvFZ5/yzqVs1K8PVkYroylSON7KhefxOZpq9tPN1UE/WaI9GQENeV7VOK+g+Su7djrIVlNowjM3owgnE8nzWqTD+RzadGy61dzwMjZUBzeYH6NOdffaLxJ3UR1V+l8p+jR62MtHiQnXtF8KRvaVL4WuE1McN98Ys+rIt0+SpvxuWBXhiwzocmEOep2VWuemacV2RBtx91b6CPaEUYKlWprsjUQLMLKJPa5jUZ5dD2LRLtiEp4zm7hrflZCMIdTCSsB5dTvWITZEgFWNTyO8wKt0CtYu4JR/AaWdMarc5/NcbTG41WiGbuByhZclXm4rCnoFnzcaqaJztKNZ6fooNVkGoS16TXZQVfmpeTZVgX7rdNmpKzc0Eo/nn0PRbpO3uspgvm+T40VWla8psWrUZgf8Mn0tO8Ca/3o8tLyoPLNEvR9YerL7Kw8nsO7PcFx/VM9k7W7pm8nXqQaudnEtvM2d+l20KK7fC6u5cZYm++Z+s1wux6bQM5qDRDd1t4OucaflZ6tnZEgTNpwsqvTqyPQkuba/rwtW/gMQNZY34lh/aJdf6R0ssi974MHjx46ld8gPEILji247FvUeD6tF89aiJu76bw6ChHeSj/D/lKO2ymSdMmnSNes1Bf+yNC01A2rK9PqmLsIe880zcgP6YcCN0dnshFVTY70ZMFau9v3xemg2se/+BePco7o=)

<img width="633" alt="image" src="https://github.com/ikun-svelte/ikun-ui/assets/8252267/50d83e0b-5994-4152-9f51-9e6d440a94bb">

**Comment by baiwusanyu-c** [comment]
狠活

**Comment by baiwusanyu-c** [comment]
可以做成loading 欸

**Comment by shabby2333** [comment]
而你，我的朋友，才是真正的英雄

**Comment by baiwusanyu-c** [comment]
后面playground的loading就用这个
